src/summary/profile/relationships-card.component.tsx

```diff
@@ -1,15 +1,55 @@
 import React from "react";
-import SummaryCard from "../cards/summary-card.component";
 import { match } from "react-router";
+import { openmrsFetch } from "@openmrs/esm-api";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+import SummaryCard from "../cards/summary-card.component";
+import SummaryCardRow from "../cards/summary-card-row.component";
+import SummaryCardRowContent from "../cards/summary-card-row-content.component";
+import VerticalLabelValue from "../cards/vertical-label-value.component";
 
 export default function RelationshipsCard(props: RelationshipsCardProps) {
+  const [relationships, setRelationships] = React.useState(null);
+  React.useEffect(() => {
+    openmrsFetch(
+      `/ws/fhir/RelatedPerson?patient.identifier=${props.patient.identifier[0].value}`
+    )
+      .then(({ data }) => {
+        if (data.total > 0) {
```

Comment:
Is this if statement needed? I think the more common thing would be to do `relationships.length > 0` on line 26, so that the emdash would still be shown if the length is zero.

Reply:
it's needed because if `total === 0` then there is no `entry` property.